packages/labs/ssr-dom-shim/src/lib/element-internals.ts

```diff
@@ -0,0 +1,141 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+// As of TypeScript 4.7.4, `ARIAMixin` is missing the following properties
+// https://w3c.github.io/aria/#state_prop_def
+declare global {
+  interface ARIAMixin {
+    ariaBraileLabel: string | null;
+    ariaBraileRoleDescription: string | null;
+    ariaDescription: string | null;
+    ariaInvalid: string | null;
+    role: string | null;
+  }
+}
+
+export const ariaMixinEnum: Record<keyof ARIAMixin, string> = {
```

Comment:
To make sure this covers all properties, I'd make a mapped type:

```ts
type ARIAAttributeMap = {
  [K in keyof ARIAMixin]: string;
}

export const ariaMixinAttributes: ARIAAttributeMap = {
```

Reply:
Done